cmd/rookd/daemon.go

```diff
@@ -47,7 +47,7 @@ func runDaemon(cmd *cobra.Command, args []string) error {
 	if err := flags.VerifyRequiredFlags(daemonCmd, []string{"type"}); err != nil {
 		return err
 	}
-	if daemonType != "mon" && daemonType != "osd" && daemonType != "mds" && daemonType != "rgw" {
+	if daemonType != "mon" && daemonType != "osd" && daemonType != "mds" && daemonType != "rgw" && daemonType != "rgw-admin" {
 		return fmt.Errorf("unknown daemon type: %s", daemonType)
```

Comment:
sounds good. I assume `tool` would be a hidden command like `daemon`. Also I think we should support non-ceph tools like `lsblk` and `sgdisk` eventually. could also make `mon mkfs` and `osd mkfs` a tool invocation.